packages/data-store-common/src/migrations/sqlite/1659463069549-CreateConnections.ts

```diff
@@ -15,7 +15,7 @@ export class CreateConnections1659463069549 implements MigrationInterface {
       `CREATE TABLE "ConnectionMetadata" ("id" varchar PRIMARY KEY NOT NULL, "label" varchar(255) NOT NULL, "value" varchar(255) NOT NULL, "connectionId" varchar)`
     )
     await queryRunner.query(
-      `CREATE TABLE "Party" ("id" varchar PRIMARY KEY NOT NULL, "name" varchar(255) NOT NULL, CONSTRAINT "UQ_Party_name" UNIQUE ("name"))`
+      `CREATE TABLE "Party" ("id" varchar PRIMARY KEY NOT NULL, "name" varchar(255) NOT NULL, "alias" varchar(255) NOT NULL, "uri" varchar(255), CONSTRAINT "UQ_Party_name" UNIQUE ("name"), CONSTRAINT "UQ_Party_alias" UNIQUE ("alias"))`
```

Comment:
URI should also be not null.

Reply:
I think there should be some thing removing these in the function down.

Reply:
incorrect. this can be null

Reply:
@BtencateSphereon any response? Is the ticket wrong or the code ?